galleon-feature-pack/src/main/resources/modules/com/microsoft/sqlserver/jre8/main/module.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module name="com.microsoft.sqlserver.jre8" xmlns="urn:jboss:module:1.8">
```

Comment:
should to be simply named `com.microsoft.sqlserver`? (jre8 seems to be an implementation detail for the version only)

Reply:
and it would be consistent with Oracle naming

Reply:
@jmesnil I reworked the mssqlserver driver name.